activesupport/lib/active_support/duration.rb

```diff
@@ -371,6 +442,7 @@ def to_s
     def to_i
       @value.to_i
     end
+    alias :to_int :to_i
```

Comment:
This one is tricky... it could be argued either way for an integer-based duration, but I think it's definitely wrong (and not currently there) for e.g. a float-based one.

Maybe we just drop it, and let the deprecation push people to an explicit conversion?